src/Terminus.php

```diff
@@ -393,6 +396,7 @@ private function addBuiltInCommandsAndHooks()
             'Pantheon\\Terminus\\Commands\\Self\\Plugin\\CreateCommand',
             'Pantheon\\Terminus\\Commands\\Self\\Plugin\\InstallCommand',
             'Pantheon\\Terminus\\Commands\\Self\\Plugin\\ListCommand',
+            'Pantheon\\Terminus\\Commands\\Self\\Plugin\\PluginBaseCommand',
```

Comment:
Same question: Do we really need to add a base class here?

Reply:
This got added via the "Add command" where it goes through and updates all the command names.